domains/neuron.json

```
@@ -0,0 +1,12 @@
+{
+  "owner": {
+    "username": "noah-dev-codes",
+    "email": "ak.no@outlook.com"
+  },
+  "records": {
+    "MX": ["mx1.improvmx.com", "mx2.improvmx.com"],
+    "TXT": ["v=spf1 include:spf.improvmx.com ~all"],
```

```suggestion
    "TXT": "v=spf1 include:spf.improvmx.com ~all",
```
If you are using single text/string in "TXT" records, you can remove the [ and ] though its not specifically required and for readability purposes.